torchfields/voting.py

```diff
@@ -27,15 +27,24 @@ def gaussian_blur(self, sigma=1, kernel_size=5):
     kernel = kernel.expand(2, 1, *kernel.shape)
     return F.conv2d(padded, weight=kernel, groups=2)
 
+def get_vote_shape(self):
```

Comment:
Since this is a rather simple one-liner, is there a reason for it not to be inline?